src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -25,14 +25,28 @@ void HexagonGame::update(ssvu::FT mFT)
 {
     mFT *= Config::getTimescale();
 
-    // TODO: refactor to avoid repetition, and truncate floating point number
-    // TODO: also show best record (here) and last run + best record (in menu)
-    steamManager.set_rich_presence_in_game(
-        levelData->name, status.getTimeSeconds());
+    // TODO: show best record (here) and last run + best record (in menu)
+
+    // Name formatter
+    std::string nameStr = std::string(levelData->name);
```

Comment:
`levelData->name` is already a string, no need to create another temporary one.

```suggestion
    std::string nameStr = levelData->name;
```